src/main/java/Pages/UserEditingPage.java

```diff
@@ -0,0 +1,124 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class UserEditingPage {
+	public UserEditingPage(WebDriver driver) {
+		PageFactory.initElements(driver, this);
+	}
+	
+	@FindBy(xpath="//i[@class='fa fa-pencil-square-o']")
+	WebElement edit_icon;
```

Comment:
across project follow any one naming convention  ( if u r using " _" follow that everywhere or use camelNotation for variables